tss-esapi/src/context.rs

```diff
@@ -56,7 +56,7 @@ pub struct Context {
     ),
     /// TCTI context handle associated with the ESYS context.
     /// As with the ESYS context, an optional Mbox wrapper allows the context to be deallocated.
-    tcti_context: TctiContext,
+    _tcti_context: TctiContext,
```

Comment:
:thinking: What's the reason for this change? I looked at #344 but I don't see any renaming... or am I looking at wrong PR?

Reply:
The main reason is "the new compilers complain about this, so I needed to make the change to get the CI to pass" 😄

Reply:
(doesn't like it because `_tcti_context` is not directly used anywhere, but we need to keep ownership to it because it's used by the ESAPI context to communicate)